tests/jeos/verify_cloudinit.pm

```diff
@@ -0,0 +1,102 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Check cloud-init configuration of the image
+# Maintainer: QA-c <qa-c@suse.com>
+
+use Mojo::Base qw(publiccloud::basetest);
+use testapi;
+use Utils::Systemd qw(systemctl);
+use Utils::Logging qw(save_and_upload_log);
+use publiccloud::ssh_interactive qw(select_host_console);
+use utils qw(ensure_serialdev_permissions);
+
+sub run {
+    my ($self, $args) = @_;
+    my @errors = ();
+    my $users = {
+        tester_ssh => {
+            passwd => 'L',
+            keyid => '~/.ssh/test_ci_ecdsa'
+        },
+        $testapi::username => {
+            passwd => 'P',
+            keyid => '~/.ssh/id_rsa'
+        }
+    };
+
+    select_console('root-console');
+    assert_script_run('cloud-init status --wait --long');
```

Comment:
shall we add a check for the output of this command?

Reply:
Returns non-zero code when the status reports a failure